.github/workflows/dev-lead-reusable.yml

```
@@ -301,3 +302,7 @@ jobs:
           echo "$payload" | gh api \
             --method POST "repos/$RELAY_REPO/dispatches" --input -
           echo "::notice::Relayed CI failure for PR $RELAY_PR_NUMBER"
+$conclusion,details_url:$details_url,app_slug:$app_slug}]}}')
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)